should install runtime header lib/tcc/include/tcclib.h

actual

```
$ (cd frida-tinycc/ && find * -type f)
include/libtcc.h
lib/libtcc.so
lib/pkgconfig/libtcc.pc
lib/tcc/include/stddef.h
lib/tcc/include/stdbool.h
lib/tcc/include/varargs.h
lib/tcc/include/stdarg.h
lib/tcc/include/stdnoreturn.h
lib/tcc/include/stdalign.h
lib/tcc/include/float.h
lib/tcc/include/ptrauth.h
```


expected

```
$ (cd tinycc/ && find lib/ include/ -regextype egrep -regex '.*\.(so|h|pc)') | grep -v lib/tcc/win32/
lib/tcc/include/stdatomic.h
lib/tcc/include/tcclib.h
lib/tcc/include/stdnoreturn.h
lib/tcc/include/stddef.h
lib/tcc/include/tgmath.h
lib/tcc/include/stdbool.h
lib/tcc/include/varargs.h
lib/tcc/include/float.h
lib/tcc/include/stdalign.h
lib/tcc/include/stdarg.h
lib/tcc/include/tccdefs.h
lib/pkgconfig/libtcc.pc
lib/libtcc.so
include/libtcc.h
```

as produced by [tinycc.nix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/compilers/tinycc/default.nix)

diff

```diff
--- actual.txt
+++ expected.txt
@@ -2,10 +2,13 @@
 lib/libtcc.so
 lib/pkgconfig/libtcc.pc
 lib/tcc/include/float.h
-lib/tcc/include/ptrauth.h
 lib/tcc/include/stdalign.h
 lib/tcc/include/stdarg.h
+lib/tcc/include/stdatomic.h
 lib/tcc/include/stdbool.h
 lib/tcc/include/stddef.h
 lib/tcc/include/stdnoreturn.h
+lib/tcc/include/tccdefs.h
+lib/tcc/include/tcclib.h
+lib/tcc/include/tgmath.h
 lib/tcc/include/varargs.h
```

at least this looks important

```
lib/tcc/include/tcclib.h
```

[tcclib.h](https://github.com/frida/tinycc/blob/main/tcclib.h) has for example `dlclose` (`error: 'dlclose' undeclared`)

[upstream](https://repo.or.cz/tinycc.git/blob/HEAD:/Makefile#l375) installs `tcclib.h` from `$(TOPSRC)/tcclib.h`

```Makefile
$(call IF,$(TOPSRC)/include/*.h $(TOPSRC)/tcclib.h,"$(tccdir)/include")
```

blame

https://github.com/frida/tinycc/blob/a438164dd4c453ae62c1224b4b7997507a388b3d/meson.build#L97-L107

> `error: 'dlclose' undeclared`

is actually an expected error in frida-gum when tcclib.h is installed

todo: fix frida-gum ...

```
[147/181] Compiling C object bindings/gumjs/libfrida-gumjs-1.0.so.p/gumcmodule.c.o
FAILED: bindings/gumjs/libfrida-gumjs-1.0.so.p/gumcmodule.c.o 
gcc -I... /build/source/build/config.h ... bindings/gumjs/libfrida-gumjs-1.0.so.p/gumcmodule.c.o ... -o bindings/gumjs/libfrida-gumjs-1.0.so.p/gumcmodule.c.o -c ../bindings/gumjs/gumcmodule.c
In file included from /nix/store/i4dqcpppyyq5yqcvw95mv5s11yfyy8pf-glib-2.74.3/lib/glib-2.0/include/glibconfig.h:9,
                 from /nix/store/7gp6sgjxwr4bzvy4r29a0rmb476pxww0-glib-2.74.3-dev/include/glib-2.0/glib/gtypes.h:34,
                 from /nix/store/7gp6sgjxwr4bzvy4r29a0rmb476pxww0-glib-2.74.3-dev/include/glib-2.0/glib/galloca.h:34,
                 from /nix/store/7gp6sgjxwr4bzvy4r29a0rmb476pxww0-glib-2.74.3-dev/include/glib-2.0/glib.h:32,
                 from ../gum/gumdefs.h:10,
                 from ../gum/gummemory.h:11,
                 from ../bindings/gumjs/gumcmodule.h:10,
                 from ../bindings/gumjs/gumcmodule.c:7:
bindings/gumjs/gumcmodule-runtime.h: In function 'gum_cmodule_get_symbols':
bindings/gumjs/gumcmodule-runtime.h:2703:70: error: 'dlclose' undeclared (first use in this function); did you mean 'pclose'?
 2703 |     g_hash_table_insert (symbols, "dlclose", GUM_FUNCPTR_TO_POINTER (dlclose));
      |                                                                      ^~~~~~~
```

codegen in [frida-gum/bindings/gumjs/generate-runtime.py](https://github.com/frida/frida-gum/blob/b82a57fa644d45cd920cc69c18520a6691556cc7/bindings/gumjs/generate-runtime.py#L297)

maybe codegen should ignore symbols from tcclib.h

--- a/meson.build
+++ b/meson.build
@@ -95,6 +95,7 @@ endif
 install_headers('libtcc.h')
 
 runtime_headers = [
+  'tcclib.h',
   'include/float.h',
   'include/ptrauth.h',
   'include/stdalign.h',

